### PR TITLE
Remove python2.7 support

### PR DESCRIPTION
**Pull request by pirlgon**
Remove pycompat.py.
Remove all sys.version testing.
Update classifier.

Use pyupgrade on temboard python file
    
find . -type d -name "node_modules" -prune -o -type f -name "*.py" -print0
| xargs -0 pyupgrade --py36-plus
in ui and agent folder.

### CODE REVIEW

File: ui/setup.py [hunk 1]
```diff
@@ -38,13 +24,11 @@
     # the user or package manager to ensure psycopg2 dependency. See
     # documentation.
     'sqlalchemy>=0.9.8,<2',
-    'tornado>=3.2,<' + BLEEDING_EDGE_TORNADO,
+    # _verify_cert in SSLIOStream is removed on Tornado 6.4+ on Python 3+.
+    'tornado>=3.2,<6.4',
```
Comment (bersace) [inline ui/setup.py:26]: On ne teste plus que Tornado 4.5 dans CentOS 7. Je propose de remplacer 3.2 par 4.5 à cette occasion.

File: agent/temboardagent/cli/register.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def main(self, args):
                 'groups': groups
             })
             response.raise_for_status()
-            dashboard_url = ui_url_raw + '/server/%s/%s/dashboard' % (
+            dashboard_url = ui_url_raw + '/server/{}/{}/dashboard'.format(
```
Comment (pgiraud) [inline agent/temboardagent/cli/register.py:161]: I think we can use f-string here as well.

File: ui/temboardui/cli/register_instance.py [hunk 3]
```diff
@@ -91,14 +91,14 @@ def main(self, args):
...
-        agent = "%s:%s" % (args.agent_address, args.agent_port)
+        agent = f"{args.agent_address}:{args.agent_port}"
 
         session = Session()
         logger.debug("Check for existing instance.")
         instance = get_instance(session, args.agent_address, args.agent_port)
         if instance:
             if not args.skip_existing:
-                raise UserError("Instance %s:%s already registered." % (
+                raise UserError("Instance {}:{} already registered.".format(
```
Comment (pgiraud) [inline ui/temboardui/cli/register_instance.py:101]: Can't we use f-string here?

File: ui/temboardui/model/orm.py [hunk 4]
```diff
@@ -303,9 +301,9 @@ def dashboard_url(self, app):
         scheme = 'https' if app.config.temboard.ssl_key_file else 'http'
         host = app.config.temboard.address
         port = app.config.temboard.port
-        path = '/server/%s/%s/dashboard' % (
+        path = '/server/{}/{}/dashboard'.format(
```
Comment (pgiraud) [inline ui/temboardui/model/orm.py:304]: Use f-string unless it takes more lines?

File: agent/temboardagent/cli/register.py (hunk 2 above)
Comment (pirlgon) [reply agent/temboardagent/cli/register.py:161]:
```
            dashboard_url = (
                f"{ui_url_raw}/server/"
                f"{args.host}/{self.app.config.temboard.port}/dashboard")
```
like this?

**Review by MarionGiusti — APPROVED** [review]

File: agent/temboardagent/cli/register.py (hunk 2 above)
Comment (pirlgon) [reply agent/temboardagent/cli/register.py]: we choose to keep format and to include ui_url_raw in it.

File: ui/temboardui/cli/register_instance.py (hunk 3 above)
Comment (pirlgon) [reply ui/temboardui/cli/register_instance.py]: better keep % formatting in this error handling

**Review by pgiraud — APPROVED** [review]

**Comment by bersace** [comment]
Vu un `__unicode__` dans services.py

**Review by bersace — APPROVED** [review]

**Pull request merged**